lib/spree/retail/engine.rb

```diff
@@ -0,0 +1,28 @@
+require 'solidus_core'
+require 'shopify_api'
+
+module Spree
+  module Retail
+    class Engine < Rails::Engine
+      isolate_namespace Spree
+      engine_name 'solidus_retail'
+
+      # TODO(cab): To remove and instead priorize a configuration initializer
```

Comment:
The Dotenv dependency should be removed sooner than later, IMO. We can live with a direct dependency on `ENV` variables, but a gem dependency is a bridge too far. Let's use [`direnv`][1] and an `.envrc` instead.

[1]: http://direnv.net/